lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -263,26 +264,13 @@ public static function customerListFunction(array $params, $template)
             if (!empty($params['selecttip'])) {
                 $result .= ' ' . self::tipFunction(array('text' => $params['selecttip']), $template);
             } else {
-                $result .= ' ' . self::tipFunction(array('text' => 'Select customer (optional)'), $template);
+                $result .= ' ' . self::tipFunction(array('text' => $firstoption . (empty($required) ? null : '(optional)')), $template);
```

Comment:
Łączenie `string` z `null`?

Reply:
przecież null się nie wyświetli zauważ ze nie jest w "" ani w ''

ale mimo to : https://github.com/chilek/lms/pull/2264/commits/3455e0b4153567e868a1b10faf86d95bb591ef86